app/components/Settings/OptionsDropdown/ESIndicesOptions.react.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {keys} from 'ramda';
+import Select from 'react-select';
+
+export default class ESIndicesOptions extends Component {
+    static propTypes = {
+        elasticsearchMappingsRequest: PropTypes.object,
+        setIndex: PropTypes.func,
+        selectedIndex: PropTypes.any
```

Comment:
If we can't use `PropTypes.string` here, then we should bring back the comment `// See type of react-select's Select 'value' property`.